controllers/admin.py

```diff
@@ -1510,6 +1511,10 @@ def add__or_update_assignment_question():
 
     autograde = request.vars.get('autograde')
     which_to_grade = request.vars.get('which_to_grade')
+    if question_type in ('activecode', 'actex'):
+        if auto_grade != 'unittest':
```

Comment:
You might add a few comments to the code here -- why is this necessary? I'm sure there's a good reason, but I don't know enough about activecode / actex problems to see why.

Reply:
Its because we never set up the defaults correctly for autograding and the picker.  We just blindly set it to % correct.  This leads to confusing scores of 0 visible to the students rather than the Not Yet Graded message.